File: Java/Human.java
public class Human
{

	//private attribute Manusia
	private String nik;
	private String nama;
	private String kelamin;

	Human(){
		//konstruktor kosong
	}

	Human(String nik, String nama, String kelamin){
		this.nik = nik;
		this.nama = nama;
		this.kelamin = kelamin;
	}


	//setter and getter//

	public void setNama(String nama) {
		// mengeset nilai nama Mahasiswa
		this.nama = nama;
	}

	public String getNama() {
		//mengembalikan nilai nama mahasiswa
		return this.nama;
	}

	public void setNik(String nik) {
		// mengeset nilai nik Mahasiswa
		this.nik = nik;
	}

	public String getNik() {
		//mengembalikan nilai nik mahasiswa
		return this.nik;
	}

	public void setKelamin(String kelamin) {
		// mengeset nilai kelamin Mahasiswa
		this.kelamin = kelamin;
	}

	public String getKelamin() {
		//mengembalikkan nilai kelamin mahasiswa
		return this.kelamin;
	}


}

File: python/Human.py
class Human:

	#private attribute Manusia
	__nik = ""
	__nama = ""
	__kelamin =""


	def __init__(self, nik="", nama="", kelamin=""):
		self.__nik = nik
		self.__nama = nama
		self.__kelamin = kelamin
	


	#setter and getter#

	def setNama(self, nama):
		# mengeset nilai nama Mahasiswa
		self.__nama = nama
	

	def getNama(self):
		#mengembalikan nilai nama mahasiswa
		return self.__nama
	

	def setNik(self, nik):
		# mengeset nilai nik Mahasiswa
		self.__nik = nik
	

	def getNik(self):
		#mengembalikan nilai nik mahasiswa
		return self.__nik
	

	def setKelamin(self, kelamin):
		# mengeset nilai kelamin Mahasiswa
		self.__kelamin = kelamin
	

	def getKelamin(self):
		#mengembalikkan nilai kelamin mahasiswa
		return self.__kelamin
	